ros_buildfarm/templates/ci/ci_job.xml.em

```diff
@@ -420,8 +420,16 @@ parameters = [
     'archive_artifacts',
     artifacts=[
       'ros%d-%s-linux-%s-%s-ci.tar.bz2' % (ros_version, rosdistro_name, os_code_name, arch),
-    ],
+    ] + show_results_images,
+))@
+@[if show_results_images]@
+@(SNIPPET(
+    'image_gallery',
+    title='Performance Test Results',
+    image_width=480,
```

Comment:
That seems fairly small. How did you choose that value?

Reply:
This was an artifact of trying to work around a bug in the plugin. It was only temporarily successful - I'm going to leave the width unrestricted instead. Good catch.